modules/custom/az_paragraphs/config/optional/editor.editor.az_standard.yml

```
@@ -114,7 +114,7 @@ settings:
         - '<dl>'
         - '<dt>'
         - '<dd>'
-        - '<div name id>'
+        - '<div class name id>'
```

```suggestion
        - '<div class name id data-*>'
```

Might be good to also add `data-*` since that exists in the text format config:
https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_media/config/install/filter.format.az_standard.yml#L31C385-L31C409

Not sure if the `*` wildcard is valid in this context, though.

---

Verified that this seems to work.